16-google/incident.ts

```diff
@@ -0,0 +1,117 @@
+/**
+ * An exploration which demonstrates packet loss and network congestion
+ * after clusters' cluster management software are de-scheduled.
+ * 
+ * This exploration exists to prove the design of the Database, Cluster 
+ * Management Software, and BGP stages appropriately mock the architecture 
+ * and problems listed in the incident report, particularly the failures 
+ * described for Google's Cloud Interconnect service.
+ * 
+ */
+
+import {
+  metronome,
+  simulation,
+  stats,
+  eventSummary,
+  stageSummary,
+  Timeout,
+  eventCompare
+} from "../../src";
+import { Cluster, Server } from "./database"
+import { ClusterManagementSoftware } from "./cms";
+import { BGP } from "./bgp";
+
+//Cluster 1
```

Comment:
These comments don't do much for understanding the incident, you can probably remove them.